.github/workflows/tests.yml

```diff
@@ -30,13 +30,19 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
 
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+
+      - uses: Swatinem/rust-cache@v1
```

Comment:
Is this working as expected?

![image](https://user-images.githubusercontent.com/8050659/150589236-e63e2f93-0fbc-4d32-9fc2-bf9eb33f0dbc.png)

Reply:
maybe because we don't have real rust repo. Let's just cache ~/.cargo even though we can't update the tool